coralproject/talk
# [CORL-2998]: update docs packages

## What does this PR do?

Keeps `docs` packages up to date.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?



## Does this PR introduce any new environment variables or feature flags?



## If any indexes were added, were they added to `INDEXES.md`?



## How do I test this PR?

`sh npm-i.sh`
`sh scripts/build-development.sh`
`cd docs`
`npm run start`
See there are no errors in the build and when you navigate to `localhost:3000`, you see the docs as expected.

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
